.github/workflows/bump-version.yml

```diff
@@ -17,26 +17,26 @@ jobs:
         id: regex-match
         with:
           text: ${{ github.event.inputs.version }}
-          regex: '^(\d+.\d+).\d+(?:-(?:(beta\d+)|(pre)))?$'
+          regex: '^(\d+.\d+).\d+(?:-(?:(preview\d?)|(pre)))?$'
       - uses: actions-ecosystem/action-regex-match@v2.0.2
         if: ${{ inputs.version_call != '' }}
         id: regex-match-version-call
         with:
           text: ${{ inputs.version_call }}
-          regex: '^(\d+.\d+).\d+(?:-(?:(beta\d+)|(pre)))?$'
+          regex: '^(\d+.\d+).\d+(?:-(?:(preview\d?)|(pre)))?$'
       - name: Validate input version
         if: ${{ steps.regex-match.outputs.match == '' && github.event.inputs.version != '' }}
         run: |
           echo "The input version format is not correct, please respect:\
-          major.minor.patch or major.minor.patch-beta.number format. \
```

Comment:
Here it says the format is `major.minor.patch-preview.number` but everywhere else the format seems to be `major.minor.patch-preview<number>`